pom.xml

```diff
@@ -1,171 +1,261 @@
+	</scm>
+
+	<distributionManagement>
+		<repository>
+			<id>artifactory.eanadev.org</id>
+			<name>artifactory.eanadev.org-releases</name>
+			<url>http://artifactory.eanadev.org/artifactory/libs-release-local</url>
+		</repository>
+		<snapshotRepository>
+			<id>artifactory.eanadev.org</id>
+			<name>artifactory.eanadev.org-snapshots</name>
+			<url>http://artifactory.eanadev.org/artifactory/libs-snapshot-local</url>
+		</snapshotRepository>
+	</distributionManagement>
+
+	<properties>
+		<version.java>13</version.java>
+		<version.springfox>2.9.2</version.springfox>
+		<version.metis>1.7.0</version.metis>
+		<version.lombok>1.18.12</version.lombok>
+		<version.commons.io>2.6</version.commons.io>
+		<version.maven.compiler.plugin>3.6.1</version.maven.compiler.plugin>
+		<version.coveralls>4.0.0</version.coveralls>
+		<version.surefire.plugin>2.22.1</version.surefire.plugin>
+
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+		<!-- Spring boot -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-amqp</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-webflux</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+
+		<!-- Europeana -->
+		<dependency>
+			<groupId>eu.europeana.metis</groupId>
+			<artifactId>metis-common</artifactId>
+			<version>${version.metis}</version>
+		</dependency>
+
+		<!-- Persistence -->
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+
+		<!-- Test -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+			<exclusions>
+				<exclusion>
+					<groupId>org.junit.vintage</groupId>
+					<artifactId>junit-vintage-engine</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>com.vaadin.external.google</groupId>
+					<artifactId>android-json</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<!-- Swagger -->
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger2</artifactId>
+			<version>${version.springfox}</version>
+		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>${version.springfox}</version>
+		</dependency>
+
+		<!-- Utils -->
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>${version.lombok}</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
IntelliJ IDE has a good plugin for Lombok, and I get your concerns, in the list example you can provide the method to return an immutable list if you want (actually I remember having that problem so I just wrote the constructor myself), so I feel it could be use as convenient, at least it helps reducing boilerplate code and provides nice features like builders. 
Anyways if we decide not to use it, I can remove it (for what I read easily)

Reply:
Same thing here, we try to avoid framework that hide too much code. In this case it's a lesser impact but with the tools we have today, getters/setters/constructors etc are very easy to generate without having to "write" that boilerplate code. Builders btw are great but from a article I read once about Lombok, don't remember where, but they were saying that they are dangerous cause you could instantiate invalid objects. 
So my opinion would be to stick away from it in general unless we collectively decide to start use it and plan for doing so everywhere.

Reply:
Sounds fair, I just wanted to give it a try, but I understand